components/build.gradle

```diff
@@ -0,0 +1,46 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+    id 'app.cash.paparazzi'
+    id 'kotlin-kapt'
+}
+
+android {
+    compileSdkVersion 33
+
+    defaultConfig {
+        minSdk 24
+        targetSdk 33
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
+    }
+    kotlinOptions {
+        jvmTarget = '11'
+    }
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion '1.4.2'
+    }
```

Comment:
out-of-scope/suggestion: I think it would be great to add resource prefix to ensure all components resources. So we're sure what resources are common. We could add prefix requirements in the build.gradle file:
```
resourcePrefix 'components_'
```

Reply:
Sounds good 👌 
I've added this to the current PR. Lines corresponding to resources were changed anyway.